orchestrationSpecs/package.json

```diff
@@ -5,9 +5,18 @@
     "packages/*"
   ],
   "dependencies": {
+    "@typescript/native-preview": "^7.0.0-dev.20251210.1",
```

Comment:
Why has this been lifted up into a general dependency?  This was only required for type-checking.